src/pages/style.sass

```diff
@@ -0,0 +1,14 @@
+
+html,body
```

Comment:
It's a good practice to have generic styles for html elements. You can try using inverted triangle architecture to achieve a clean implementation tho